backend/src/db/users.js

```diff
@@ -1,45 +1,72 @@
 const models = require('../models');
 const db = require('../sequalize_pg');
 
-module.exports = {
-    createUser: async (user) => {
-        await db.sync();
-        return await models.users.create(user);
-    },
+const createUser = async (user) => {
+    await models.users.create(user);
+    return await getUser(user.phoneNumber);
+};
 
-    getUser: async (phoneNumber) => {
-        return await models.users.find({
-            where: { phoneNumber },
-        });
-    },
-
-    getUserByPhoneAndPin: async (phoneNumber, pin) => {
-        return await models.users.find({
-            where: { phoneNumber, pin },
-        });
-    },
-
-    getUserByToken: async (token) => {
-        return await models.users.find({
-            where: { token },
-        });
-    },
-
-    updateUser: async (user, phoneNumber) => {
-        return await models.users.update(
-            {
-                pin: user.pin,
-                username: user.username,
-            },
-            {
-                where: { phoneNumber },
-            }
-        );
-    },
-
-    deleteUser: async (phoneNumber) => {
-        return await models.users.destroy({
+const getUser = async (phoneNumber) => {
+    return await models.users.find({
+        where: { phoneNumber },
+    });
+};
+
+const getUserByPhoneAndPin = async (phoneNumber, pin) => {
+    return await models.users.find({
+        where: { phoneNumber, pin },
+    });
+};
+
+const getUserByToken = async (token) => {
+    return await models.users.find({
+        where: { token },
+    });
+};
+
+const updateUser = async (user, phoneNumber) => {
+    return await models.users.update(
+        {
+            pin: user.pin,
+            username: user.username,
+        },
+        {
             where: { phoneNumber },
-        });
-    },
+        }
+    );
+};
+
+const getUserBalance = async (userId) => {
```

Comment:
This method should return the balance, not the user object. You could apply a `then` block like this

```
await models.users.find(...).then(user => user.balance)
```